### ISSUE TITLE
Editorial - "Reject" decision in PhysioNet Editorial disables project content access

### ISSUE THREAD

**Issue by lilehman** [MIT-LCP/physionet-build#2116]
Please allow contributors  to (at least temporarily) access their project content after their project has been Rejected.

**Comment by bemoody** [comment]
I think that the separation between "active", "archived", and "published" projects was a mistake.  I think it would be best to have "archived" (split into "rejected" and "withdrawn") and "published" as additional *submission states* for the project.  That would be a major change to the system but would have a lot of benefits.

I do also think there should be a strict time limit (30 days?) after which time the project content is erased.  If folks want a backup, back it up somewhere else.

**Comment by tompollard** [comment]
> I think that the separation between "active", "archived", and "published" projects was a mistake. I think it would be best to have "archived" (split into "rejected" and "withdrawn") and "published" as additional submission states for the project. 

I agree that our current approach is not optimal! I'm not sure I understand your suggested improvement...we should discuss this at some point.

**Comment by bemoody** [comment]
Yeah was not clear.  The *fundamental reason* that you can't view
rejected projects (or do anything else with them) is that rejecting
causes the project to be converted from an ActiveProject object into
an ArchivedProject.

And if we wanted to let authors/editors view ArchivedProjects the way
they can view ActiveProjects, we'd have to write a bunch of new view
functions or else "genericize" the existing ones, either of which
would be a pain to implement and maintain.
